ts/components/screens/BaseScreenComponent/utils.tsx

```diff
@@ -40,24 +38,16 @@ export const handleOnLinkClicked = (hideHelp: () => void) => (url: string) => {
  */
 export const getContextualHelpConfig = (
   contextualHelp: ContextualHelpProps | undefined,
-  contextualHelpMarkdown: ContextualHelpPropsMarkdown | undefined,
-  onLoadEnd: () => void,
-  onLinkClicked: (url: string) => void,
-  shouldHandleLink?: (url: string) => boolean
+  contextualHelpMarkdown: ContextualHelpPropsMarkdown | undefined
+  // onLoadEnd: () => void,
+  // onLinkClicked: (url: string) => void,
+  // shouldHandleLink?: (url: string) => boolean
```

Comment:
What about removing this comments? What do you think?

Reply:
I wasn't sure if we could get rid of these methods, so I left them in. They were managed by the legacy Markdown component, but I'm not sure if they are still needed or not.